test/cql-pytest/cassandra_tests/validation/entities/json_test.py

```diff
@@ -258,10 +258,12 @@ def testFromJsonFct(cql, test_keyspace):
             assert_rows(execute(cql, table, "SELECT k, booleanval FROM %s WHERE k = ?", 0), [0, False])
 
             # strings are also accepted
-            # Reproduces issue #7915
```

Comment:
again, please don't remove this comment. This is a comment which I added to say exactly which part of the very long test (this is what the original Cassandra test looked like) reproduced this issue. I think this information is still useful after the issue is fixed - in case one day we have a regression or something.